CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
 
 </details>
 
+## 0.98.0 (May 22, 2019)
+
+### Minor
+
+- Icon: Add new icon of `heart-outline` and update icon of `heart` (#526)
```

Comment:
this would go under the `unreleased` section above, no need to add the `0.98` title yet